fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/InviteFragment.kt

```diff
@@ -73,6 +79,35 @@ class InviteFragment : Fragment() {
     LaoActivity.addBackNavigationCallbackToEvents(requireActivity(), viewLifecycleOwner, TAG)
   }
 
+  private fun setupCopyServerButton(binding: InviteFragmentBinding) {
+    val serverTextView = binding.laoPropertiesServerText
+    val copyButton = binding.copyServerButton
+
+    copyButton.setOnClickListener {
+      val text = serverTextView.text.toString()
+      copyTextToClipboard(text)
+      Toast.makeText(requireContext(), R.string.successful_copy, Toast.LENGTH_SHORT).show()
+    }
+  }
+
+  private fun setupCopyIdentifierButton(binding: InviteFragmentBinding) {
+    val identifierTextView = binding.laoPropertiesIdentifierText
+    val copyButton = binding.copyIdentifierButton
+
+    copyButton.setOnClickListener {
+      val text = identifierTextView.text.toString()
+      copyTextToClipboard(text)
+      Toast.makeText(requireContext(), R.string.successful_copy, Toast.LENGTH_SHORT).show()
+    }
+  }
```

Comment:
Refactoring opportunity: these 2 methods are almost the same, so you can extract a single method, which is more generic, and parametrise its usage to reduce code duplication